lnst/Recipes/ENRT/DoubleBondRecipe.py

```diff
@@ -118,3 +119,7 @@ def dev_interrupt_hw_config_dev_list(self):
     def parallel_stream_qdisc_hw_config_dev_list(self):
         host1, host2 = self.matched.host1, self.matched.host2
         return [host1.eth0, host1.eth1, host2.eth0, host2.eth1]
+
+    @property
+    def vf_trust_device_list(self) -> list[RemoteDevice]:
+        return [sriov_devices.phys_dev for sriov_devices_list in self.vf_config.values() for sriov_devices in sriov_devices_list]
```

Comment:
the nested loop returns a single device i think so probably better to name the variable `sriov_device` instead of `sriov_devices` right?

Reply:
The class name is `SRIOVDevices`, so I followed this pattern. I thought of SRIOVDevices class as a collection of VFs over a PF.

The `self.vf_config.values()` returns `list[list[SRIOVDevices]]`.

Reply:
> The self.vf_config.values() returns list[list[SRIOVDevices]].

this is why i'm confused about this `vf_trust_device_list` property, in some recipes you have two nested loops and in some just one loop...

Reply:
I changed this so that the individual recipes return list of the vf_devices (the remapped device requirements) and the UseVfsMixin that has access to SRIOVDevices structures would gather the PF devices.